firedrake/slate/slac/compiler.py

```diff
@@ -100,7 +103,7 @@ def compile_expression(slate_expr, tsfc_parameters=None, coffee=False):
     :class:`firedrake.op2.Kernel` object representing the Slate expression.
 
     :arg slate_expr: a :class:'TensorBase' expression.
-    :arg tsfc_parameters: an optional `dict` of form compiler parameters to
+    :arg parameters: an optional `dict` of form compiler parameters to
```

Comment:
I don't think you meant to change this. Or if you did, then you should change the code to match.